tensorflow/c/experimental/gradients/BUILD

```diff
@@ -1,5 +1,21 @@
 load("//tensorflow:tensorflow.bzl", "filegroup")
 load("//tensorflow/core/platform:rules_cc.bzl", "cc_library")
+load(
+    "//tensorflow:tensorflow.bzl",
+    "if_libtpu",
```

Comment:
Loaded symbol "if_libtpu" is unused. Please remove it.

Reply:
Done